### PR TITLE
docs: build.zh.md, lint.zh.md chinese version, issue 710

### LINKED ISSUE #710: :open_book: `Docs(Translation)`: Build & Lint - Chinese Version
### What should be changed?

Now we have the document [Build](https://github.com/devstream-io/devstream/blob/main/docs/development/build.md) and [Lint](https://github.com/devstream-io/devstream/blob/main/docs/development/lint.md), so the corresponding Chinese version can also start writing (translation) [here](https://github.com/devstream-io/devstream/blob/main/docs/development/build.zh.md) and [here](https://github.com/devstream-io/devstream/blob/main/docs/development/lint.zh.md).

---

The translation does not need to correspond strictly to each sentence, it just needs to express the same meaning. "Meaningful translation" is recommended . Please feel free to ask me when you have any questions.

### PR DESCRIPTION
**Pull request by wendao365**
## Pre-Checklist

Note: please complete **_ALL_** items in the following checklist.

- [x] I have read through the [CONTRIBUTING.md](https://github.com/devstream-io/devstream/blob/main/CONTRIBUTING.md) documentation.
- [ ] My code has the necessary comments and documentation (if needed).
- [ ] I have added relevant tests

## Related Issues
close #710

### CODE REVIEW

**Comment by aFlyBird0** [comment]
Hi @wendao365 

Thanks for your contribution! We will test your change locally and try to merge it as soon as possible.

Please join our [Slack here](https://join.slack.com/t/devstream-io/shared_invite/zt-16tb0iwzr-krcFGYRN7~Vv1suGZjdv4w). Feel free to reach out and request to join the [contributor channel](https://devstream-io.slack.com/messages/contributors)! We are also in CNCF now; please also consider joining [this channel in CNCF Slack](https://cloud-native.slack.com/messages/devstream).

In the meantime, feel free to check out our [new contributor guide](https://docs.devstream.io/en/latest/contributing_guide/#new-contributor-guide). We have quite a few pieces of documentation explaining the basic concepts and how to work on DevStream. Read them in the ["Developer Guide"](https://docs.devstream.io/en/latest/development/development-workflow/) section of the documentation.

If you are searching for something else to work on, please do check out the ["find an issue"](https://docs.devstream.io/en/latest/contributing_guide/#find-an-issue) section. The ["help wanted"](https://github.com/devstream-io/devstream/labels/help%20wanted) label is an excellent place to continue. If you are not completely confident yet, maybe you can warm yourself up by solving another ["good first issue"](https://docs.devstream.io/en/latest/development/good-first-issues/).

**Comment by aFlyBird0** [comment]
Your commit message follows [conventional commits specification](https://www.conventionalcommits.org/en/v1.0.0/), I think it is great! 👍

**Comment by aFlyBird0** [comment]
A kind reminder that you haven't signed off on your commit(s) yet.

Check out our [official doc here](https://docs.devstream.io/en/latest/contributing_guide/#sign-your-commits) to learn how to do this.

Licensing is important to open source projects. It provides some assurances that the software will continue to be available based on the terms that the author(s) desired.

Now that we are part of CNCF, per CNCF's requirement, we kindly require that contributors sign off on commits submitted to our project's repositories. Thanks!

File: docs/development/lint.zh.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
-# Lint
+# 代码检查
 
-todo
+
+项目使用`golangci-lint` ([official website](https://golangci-lint.run/), [GitHub](https://github.com/golangci/golangci-lint)) 做代码检查，也是Go的代码检查工具集。
+
+也支持与 Github Actions 工作流集成。
```
Comment (aFlyBird0) [inline docs/development/lint.zh.md:6]: GitHub

File: docs/development/lint.zh.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
-# Lint
+# 代码检查
 
-todo
+
+项目使用`golangci-lint` ([official website](https://golangci-lint.run/), [GitHub](https://github.com/golangci/golangci-lint)) 做代码检查，也是Go的代码检查工具集。
```
Comment (aFlyBird0) [inline docs/development/lint.zh.md:4]: Maybe: ”项目使用 Go 的代码检查工具集 `golangci-lint` (link1 link2) 进行代码格式检查。“ is more fluent?

File: docs/development/lint.zh.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+
+也支持与 Github Actions 工作流集成。
+
+- 记录默认启用的检查项的 [文档](https://golangci-lint.run/usage/linters/) 。
+- 也可以和 [IDE](https://golangci-lint.run/usage/integrations/) 集成。
+- 也可以在 [本地运行](https://golangci-lint.run/usage/quick-start/) 。
+
+除此之外，我们使用 [Go Report Card](https://goreportcard.com/report/github.com/devstream-io/devstream) ，为你的开源 Go 代码生成质量报告。
+
+在项目的README.md中，有代码质量标志 [![Go Report Card](https://goreportcard.com/badge/github.com/devstream-io/devstream)](https://goreportcard.com/report/github.com/devstream-io/devstream) 。
```
Comment (aFlyBird0) [inline docs/development/lint.zh.md:14]: Maybe "代码质量报告"? "标志" is a little strange.

Comment (aFlyBird0) [reply docs/development/lint.zh.md:14]: Maybe we need not to translate "Go Report Card".

Just: "除此之外，我们使用 [Go Report Card]() 生成代码质量报告。"

What's your opinion? @wendao365

Comment (aFlyBird0) [reply docs/development/lint.zh.md:14]: Maybe we need not to translate "Go Report Card".

Just: "除此之外，我们使用 [Go Report Card](https://github.com/devstream-io/devstream/pull/739) 生成代码质量报告。"

What's your opinion? @wendao365

Comment (aFlyBird0) [inline docs/development/lint.zh.md:14]: Good!

File: docs/development/build.zh.md [hunk 4]
```diff
@@ -1 +1,17 @@
...
+make build -j8 # 多线程构建
+```
+
+这是进行全局构建：`dtm`和所有的插件
+
+项目也支持以下构建模式：
+
+- 只构建 `dtm`：`make build-core`。
+- 构建指定的插件：`make build-plugin.PLUGIN_NAME`。例如： `make build-plugin.argocd`。
+- 构建所有插件：`make build-plugins -j8` (多线程构建。)
```
[the remark this answers is not in the document]
Comment (wendao365) [reply docs/development/build.zh.md:15]: Thanks for your suggestion, I think it's correct, I will revise it soon

File: docs/development/build.zh.md [hunk 5]
```diff
@@ -1 +1,17 @@
 # 构建
+
+```shell
+cd path/to/devstream
+make clean
+make build -j8 # 多线程构建
+```
+
+这是进行全局构建：`dtm`和所有的插件
```
[the remark this answers is not in the document]
Comment (wendao365) [reply docs/development/build.zh.md:9]: very good，more semantically and comprehensible

File: docs/development/lint.zh.md (hunk 1 above)
Comment (wendao365) [reply docs/development/lint.zh.md:6]: OK，I will replace

File: docs/development/lint.zh.md (hunk 2 above)
Comment (wendao365) [reply docs/development/lint.zh.md:4]: yes，should use this translation

File: docs/development/lint.zh.md (hunk 3 above)
Comment (aFlyBird0) [reply docs/development/lint.zh.md:14]: > I will submit as soon as possible after the modification is complete, should I submit a new pr

No, you  needn't. Just git push more commits and the new commits will be added to this pr.

**Comment by IronCore864** [comment]
@aFlyBird0 please help the original contributor to merge this PR as soon as possible. We can squash and update the commit messages ourselves for first-time contributors.

**Review by aFlyBird0 — APPROVED** [review]

**Comment by aFlyBird0** [comment]
Could you please review this pr? Thank you! @imxw

**Review by IronCore864 — APPROVED** [review]

**Pull request merged**